draft-ietf-tls-iana-registry-updates.md

```diff
@@ -223,6 +221,12 @@ TLS_DHE_PSK_WITH_CHACHA20_POLY1305_SHA256     | {0xCC,0xAD}
 
     Note(3): The designated expert {{RFC5226}} only ensures that the specification is publicly available.
 
+TLS Supported Groups
+====================
+
+Add a “Recommended” column with a "Yes" for secp256r1, secp384r1, and ecdh_x25519 while all others are "No".  These "Yes" groups are taken from Standards Track RFCs.  Note that all groups from {{I-D.ietf-tls-rfc4492bis}}, which is standards track, are not marked as "Yes"; only those that are musts or shoulds from {{I-D.ietf-tls-tls13}} are marked as "Yes"; these groups also apply to previous version of TLS {{I-D.ietf-tls-rfc4492bis}}.  A Standards Track document {{RFC5226}} is required to register an entry  with the value "Yes".
+
```

Comment:
Also, I tend to think we should make 448 a Yes, though it breaks your clean rule